devops/resources/remove-data-czar.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -exuo pipefail
+
+set +u
+. /edx/var/jenkins/jobvenvs/virtualenv_tools.sh
+# creates a venv with its location stored in variable "venvpath"
+create_virtualenv --python=python3.8 --clear
+. "$venvpath/bin/activate"
+set -u
+
+cd "$WORKSPACE/configuration"
+pip install -r util/jenkins/requirements.txt
+. util/jenkins/assume-role.sh
+
+assume-role ${ROLE_ARN}
+
+cd util/create_data_czar
```

Comment:
I guess, for the better naming convention we can use the data_czar folder in the configuration repo. That folder should have `create_data_czar` and `delete_data_czar` scripts. Once you land that change you may need to change these paths/names